apps/emqx_management/src/emqx_mgmt_api_clients.erl

```diff
@@ -250,7 +250,15 @@ parse_ratelimit_str(S) ->
 %% Format
 
 format(Items) when is_list(Items) ->
-    [format(Item) || Item <- Items];
+    lists:foldr(
+      fun(Item, Acc) ->
+        try
+            [format(Item) | Acc]
+        catch
+            throw:gone:_Stk ->
```

Comment:
oh, here is the catch.
this is quite an implicit cross-module API. (yes, catching exception like this is considered an API).

maybe change the `item/1` call to return  `{ok, Result} | {error, Reason}` or maybe `map() | false`
and add a spec for `item/1`. which I believe is better than an exception driven design.